io.itch.tx00100xt.SeriousSamClassic-VK.yaml

```diff
@@ -0,0 +1,46 @@
+app-id: io.itch.tx00100xt.SeriousSamClassic-VK
+runtime: org.freedesktop.Platform
+sdk: org.freedesktop.Sdk
+runtime-version: '23.08'
+command: serioussam
+
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --device=dri
+  - --share=ipc
+  - --share=network
+  - --socket=pulseaudio
+
+cleanup:
+  - '*.la'
+  - '*.a'
+
+modules:
+
+  - name: serioussam
+    buildsystem: cmake-ninja
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=RelWithDebInfo
+    sources:
+      - type: archive
+        url: https://github.com/tx00100xt/SeriousSamClassic-VK/archive/refs/tags/1.10.6e.tar.gz
+        sha256: aec1d1ef3a16bb914d378d8b777d8178341253994d0e8080a514a67814039bce
+        x-checker-data:
+          type: anitya
+          project-id: 9154
+          url-template: https://github.com/tx00100xt/SeriousSamClassic-VK/archive/refs/tags/$version.tar.gz
+      - type: patch
+        path: 0001-CMakeLists.txt.patch
+      - type: patch
+        path: 0002-Engine.cpp.patch
+      - type: patch
+        path: 0003-desktop-file.patch
+
+  - name: metainfo
+    buildsystem: simple
+    sources:
+      - type: file
+        path: io.itch.tx00100xt.SeriousSamClassic-VK.metainfo.xml
+    build-commands:
+      - install -Dm 644 io.itch.tx00100xt.SeriousSamClassic-VK.metainfo.xml -t /app/share/metainfo
```

Comment:
```suggestion
    post-install:
      - install -Dm 644 io.itch.tx00100xt.SeriousSamClassic-VK.metainfo.xml -t /app/share/metainfo
    sources:
      - type: archive
        url: https://github.com/tx00100xt/SeriousSamClassic-VK/archive/refs/tags/1.10.6e.tar.gz
        sha256: aec1d1ef3a16bb914d378d8b777d8178341253994d0e8080a514a67814039bce
        x-checker-data:
          type: anitya
          project-id: 9154
          url-template: https://github.com/tx00100xt/SeriousSamClassic-VK/archive/refs/tags/$version.tar.gz
      - type: patch
        path: 0001-CMakeLists.txt.patch
      - type: patch
        path: 0002-Engine.cpp.patch
      - type: patch
        path: 0003-desktop-file.patch
      - type: file
        path: io.itch.tx00100xt.SeriousSamClassic-VK.metainfo.xml
```

Reply:
also I doubt the project-id is correct.

![Screenshot from 2024-01-25 12-13-15](https://github.com/flathub/flathub/assets/114441/e1c0184b-dee7-4bee-a964-1253432158f9)